grid_helper.js

```diff
@@ -627,7 +608,8 @@ function buildView(divId, initialTableau, numCols=12, gridRowHeight=200) {
             that.locationHash = '';
             window.location.hash = '';
             that.$nextTick(function() {
-              applySmartdown(tableau, function() {
+              var loadedTableauCells = tableau.layout.cells;
+              applySmartdownRecursively(tableau, function() {
```

Comment:
First argument to `applySmartdownRecursively()` should be loadedTableauCells instead of tableau.